.github/workflows/update-flakes.yml

```diff
@@ -24,7 +24,7 @@ jobs:
         uses: actions/checkout@v3
 
       - name: Run update-flakes
-        run: ./update-flakes.sh
+        run: ./update-flake.sh
```

Comment:
confusing. one is flakes.yml and one is flake.sh

Reply:
Fixed.